### DISCUSSION TITLE
What files do I download?

### DISCUSSION THREAD

**Discussion by DramaticCheese** [publicsamples/Oi-Grandad#30] [Q&A]
I've never actually used github so i was just wonder what files to download (also this plugin looks rlly cool)

**Comment by RickTaylor79** [comment]
No worries, I think everyone has that problem to begin with.  You can download the plugin here:

**https://github.com/publicsamples/Oi-Grandad/releases/tag/1.1.4**

Typically on GitHub the main repository contains source code while executable apps and so are found on the release page.
